src/commands/rsync.ts

```diff
@@ -20,10 +42,29 @@ export const rsyncCommand = async () => {
         placeHolder: "Which plugin would you like to sync?",
     });
 
-    const wpPath = await vscode.window.showInputBox({
-        prompt: 'Enter the remote path to upload the plugin contents to.',
-        placeHolder: 'user@server:public_html/wp-content/plugins/jetpack',
-    });
+	const existingSite = await vscode.window.showQuickPick(getSavedSites(configFilePath), {
+		placeHolder: "Which site would you like to sync to?"
+	});
+
+	if (typeof existingSite === 'undefined') {
+		console.log('No site selected, exiting.');
+		return;
+	}
+
+	let wpPath: string | undefined;
+	if (existingSite === '+ Add New Site') {
```

Comment:
Maybe `+ Add New Site` should be a constant at the top of the file or something. I think this could end up being very error prone because it looks like a plain string in a lot of places and it might not be obvious we're running comparisons against it.

Reply:
Brilliant, fixed!